src/Neo.Cryptography.BLS12_381/ConstantTimeUtility.cs

```diff
@@ -26,7 +26,7 @@ public static bool ConstantTimeEq<T>(in T a, in T b) where T : unmanaged
         for (int i = 0; i < a_u64.Length; i++)
             f |= a_u64[i] ^ b_u64[i];
         for (int i = a_u64.Length * sizeof(ulong); i < a_bytes.Length; i++)
-            f |= (ulong)a_bytes[i] ^ a_bytes[i];
+            f |= (ulong)a_bytes[i] ^ b_bytes[i];
```

Comment:
Quite a straightforward fix, just a question from my side: is it possible that `b_bytes` has the length less than `a_bytes`? We're iterating up to `a_bytes.Length`, so this code is valid if these spans have the same length.

Reply:
Yes we need to check the length before

Reply:
We actually don't, thanks to Shili I've read a bit of documentation :D